src/panels/ActionsPanel/ActionsPanel.tsx

```diff
@@ -211,12 +244,21 @@ export default function ActionsPanel(): JSX.Element {
               cancelButtonLabel={Label.CANCEL_BUTTON}
               confirmButtonLabel={Label.CONFIRM_BUTTON}
               confirmButtonAppearance="positive"
-              onConfirm={() => {
+              onConfirm={(event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+                event.nativeEvent.stopImmediatePropagation();
```

Comment:
This line of code (and maybe similar lines in other places) might become redundant in the case [this gets implemented](https://github.com/canonical/react-components/issues/1032).

Reply:
Thanks for filing that issue. I think it would be good to leave a comment here about why this `stopImmediatePropagation` is necessary and probably link to the issue you filed as well.